src/persistence/settingsserializer.cpp

```diff
@@ -412,11 +411,9 @@ void SettingsSerializer::readIni()
         // Add all keys
         if (!s.group().isEmpty())
             beginGroup(s.group());
+
         for (QString k : s.childKeys())
-        {
             setValue(k, s.value(k));
-            //qDebug() << "Read key "<<k<<" in group "<<group<<":\""<<groups[group]<<"\"";
-        }
```

Comment:
Yeah, and especially on loops, this easily leads to irritations and error prone code. I prioritize avoidance of confusion rather than following a dumb style guide! Please @tux3, @zetok: Can we change (actually correct) the "bracket rules" for loops please and make brackets mandatory there? It's fine with "if" statements, yet I tend to put brackets there as well. The problem is, after you followed lots of (different) style guides, you finally learn, that you don't need one. Fortunately that "tab against white space" war is finally over! :)

Btw. @Diadlo A single line `switch` statement doesn't make any sense. So brackets go there "automagically"… :)